### PR TITLE
XW-181 | Do not allow nulls from datamart table

### PR DESCRIPTION
**Pull request by vforge**
@Warkot

### CODE REVIEW

**Comment by bognix** [comment]
LGTM

File: includes/wikia/services/WAMService.class.php [hunk 1]
```diff
@@ -319,6 +319,8 @@ protected function getWamIndexConditions ($options, $db) {
 			}
 		}
 
+		$conds[] = '(dw.url IS NOT NULL AND dw.title IS NOT NULL)';
```
Comment (Warkot) [inline includes/wikia/services/WAMService.class.php:322]: Are the parentheses really needed here?

Comment (vforge) [inline includes/wikia/services/WAMService.class.php:322]: Nope, but it's logical separation because of context

Comment (Warkot) [inline includes/wikia/services/WAMService.class.php:322]: OK, sounds reasonable.

**Comment by Warkot** [comment]
LGTM :+1:

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**